submissions/asaMitaka/hooliStylePopup/style.css

```diff
@@ -0,0 +1,182 @@
+.nav__ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: end;
+}
+
+.nav__item {
+    padding: 0 10px;
+    list-style: none;
+    display: flex;
+    align-items: center;
+}
+
+.nav__item a {
+    color: black;
+    text-decoration: none;
+}
+
+.nav__item a:hover {
+    color: rgb(78, 69, 69);
+    text-decoration: underline;
+}
+
+:focus {
+    outline: 2px solid blue;
+}
+
+input[type='checkbox']:focus + label {
```

Comment:
Please, avoid styling by tags at all, use class names [link](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)